### DISCUSSION TITLE
Result query set expired error occurs in Hue with k8s

### DISCUSSION THREAD

**Discussion by hoon-e** [cloudera/hue#2536] [Questions] [answered]
# Bug
I currently use hue application on kubernetes cluster.
I connect to hive session using beeswax. 
beesawx attribute values are as follows.

```
[beeswax]
  hive_server_host=< hiveserver host >
  hive_server_port=10000
  hive_conf_dir=< hive conf directory >
  server_conn_timeout=12000
  download_row_limit=10000000
  download_cell_limit=10000000
  download_bytes_limit=350000000
  close_queries=true
  max_number_of_sessions=300
```

When i executed hive query, the following screen is repeated, and the result is that the query result is unsuccessful.

# Picture
![Screen Shot 2021-09-01 at 1 28 44 PM](https://user-images.githubusercontent.com/32868349/131612460-b5d8921c-c1f5-4aff-88de-e1de8ccfdb5e.png)
![Screen Shot 2021-09-01 at 1 28 37 PM](https://user-images.githubusercontent.com/32868349/131612478-90ca4fd1-0d43-4a48-ba3f-7fbaec4fe248.png)

network status is like below. It seems to keep checking the status.
I wonder if it has something to do with readinessProbe.
![screenshot-1](https://user-images.githubusercontent.com/32868349/131619371-00bfd681-50c4-48d1-b1b5-1a110c2d7695.png)

Have you ever experienced a similar situation?
Thanks.

# Hue Version
Opensource 4.9.0

**Comment by romainr** [comment]
Are you connections to HS2 sticky? Same for you users if using multiple Hue

https://docs.gethue.com/administrator/administration/reference/#impala-and-hive-ha

**Comment by hoon-e** [comment]
Yes. I use HS2 sticky.
Is it a sticky session issue? I wonder if it's a load balancing problem, so even if I change to a single host, the same error occurs.

**Comment by hoon-e** [comment]
https://gethue.com/how-to-optimally-configure-your-analytic-database-for-high-availability-with-hue-and-other-sql-clients/

I've been testing separating property values for hue according to the documentation.

**Comment by hoon-e** [comment, marked as answer]
It works!
As shown in the figure, I separated the haproxy port and configured it, and it worked properly.

One thing to note is that the following error occurred,
```
TSocket read 0 bytes (code THRIFTTRANSPORT): TTransportException('TSocket read 0 bytes',)
```

It was an error that occurred because the SSL setting was added to the server while setting the new port.
If this error occurs, you may check the following.
1. Check if SSL settings are applied to the server.
2. Make sure SSL is turned off in hue hiveserver or Impala beeswax settings.
3. If set, check if the SSL key file is set properly.

**I have all the SSL settings turned off in Hue I run.**

**Reply by KeithTt** [reply]
Hi, what does `separated the haproxy port ` mean?

If I use Nginx, how to separate?
